types/recharts/index.d.ts

```diff
@@ -96,6 +98,7 @@ export interface CategoricalChartWrapper<L = LayoutType> {
     barCategoryGap?: number | string;
     barGap?: number | string;
     barSize?: number | string;
+    baseValue?: BaseValueType;
```

Comment:
I'm not very familiar the library but based on the documentation, seems like `baseValue` is only specific of `AreaChart`. Adding it to the interface will add it to another bunch of classes where the property is not documented. Is that the intended behavior?

Reply:
Because you can put  `Area` inside of composed charts, the prop needs to be there as well so it can be passed down to children of `ComposedChart`.   For a real-world example, if you go to their Composed Char example: http://recharts.org/en-US/examples/LineBarAreaComposedChart you can add `baseValue={450}` to the  `ComposedChart` component and it will have the desired effect. So I'm just documenting existing behavior with this change.

Reply:
Thanks for clarifying.